src/test/functional/sessions.test.ts

```diff
@@ -0,0 +1,80 @@
+import { TestUtil } from "./testUtil";
+import axios from "axios";
+import chai from "chai";
+
+const testUtil = new TestUtil();
+
+const frontendURL = process.env.TEST_URL || "http://localhost:8080";
+const username = "b@a.com";
+const password = "4590fgvhbfgbDdffm3lk4j";
```

Comment:
maybe put these credentials in the vault and get them like we do for the redis-password
I mean I'm not really sure if it's necessary, but if it's not too much hassle, it saves us the headache